libs/librepcb/project/schematics/graphicsitems/sgi_symbol.cpp

```diff
@@ -77,12 +83,36 @@ void SGI_Symbol::updateCacheAndRepaint() noexcept
 
     // polygons
     for (const Polygon& polygon : mLibSymbol.getPolygons()) {
+        // get the bounding rectangle for the polygon
```

Comment:
This comment is misleading as the next line does *not* get the bounding rect, it gets the exact outline. Better no comment than a wrong one :wink: (isn't the code self-explaining anyway?)

Reply:
True, I'll fix that.

> isn't the code self-explaining anyway?

It took me a while to understand what it does. Especially if you're not familiar with the Qt graphics APIs. And even if the code is easy to understand, I think comments help to logically group multiple statements. But opinions on that vary :slightly_smiling_face:

Reply:
Fixed the comment.